flytepropeller/pkg/resolver/k8s_resolver.go

```diff
@@ -0,0 +1,182 @@
+package resolver
+
+import (
+	"context"
+	"fmt"
+	"github.com/flyteorg/flyte/flytestdlib/logger"
+	"google.golang.org/grpc/grpclog"
+	"google.golang.org/grpc/resolver"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"net"
+	"strings"
+	"sync"
+)
+
+const (
+	KubernetesSchema = "k8s"
```

Comment:
Should this be `kubernetes` as your comment shows?

Reply:
sure